From b0b453e4e70073d63aebe6bb259f4c891ba08c21 Mon Sep 17 00:00:00 2001
From: Nicolas Chauvet (kwizart) <kwizart@gmail.com>
Date: Wed, 13 May 2009 11:17:44 +0200
Subject: [PATCH] Fix missing symbol in libxvmc_plugin.so

the XvMC plugin use xcommon.c where some part use
HAVE_XINERAMA conditions.

Missing symbols are :
XineramaQueryScreens
XineramaQueryExtension
XineramaIsActive
---
 configure.ac |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index be6e24c..414a29d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4106,12 +4106,14 @@ AS_IF([test "$enable_xinerama" != "no"], [
       VLC_ADD_LIBS([xvideo],[-lXinerama_pic])
       VLC_ADD_LIBS([x11],[-lXinerama_pic])
       VLC_ADD_LIBS([glx],[-lXinerama_pic])
+      VLC_ADD_LIBS([xvmc],[-lXinerama_pic])
       ac_cv_have_xinerama="yes"
     ],[
       AC_CHECK_LIB(Xinerama, XineramaQueryExtension,[
         VLC_ADD_LIBS([xvideo],[-lXinerama])
         VLC_ADD_LIBS([x11],[-lXinerama])
         VLC_ADD_LIBS([glx],[-lXinerama])
+        VLC_ADD_LIBS([xvmc],[-lXinerama])
         ac_cv_have_xinerama="yes"
       ])
     ])
-- 
1.6.0.6

